docs/src/main/asciidoc/deploying-to-openshift-howto.adoc

```diff
@@ -0,0 +1,109 @@
+////
+This document is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+////
+TODO:
+- Title: https://quarkus.io/guides/doc-reference#titles-headings
+- Use the file name as the ID
+- Choose appropriate categories: https://quarkus.io/guides/doc-reference#categories
+////
+[id="deploying-to-openshift-howto"]
+= Deploy {project-name} Java applications to OpenShift
+include::_attributes.adoc[]
+:diataxis-type: howto
+:categories: cloud, native
+:summary: This guide covers how to build and deploy a Quarkus application on OpenShift in a single step.
+:topics: devops,kubernetes,openshift,cloud,deployment
+:extensions: io.quarkus:quarkus-openshift
+
+As an application developer, you can deploy your {project-name} applications to OpenShift by using a single Maven command.
+
+[NOTE]
+====
+To run this deployment, it is not necessary to have the `quarkus-openshift` extension included in your {project-name} project.
+====
+
+== Prerequisites
+
+* You have OpenJDK 17 or later installed.
+* You have set the `JAVA_HOME` environment variable to the location of the Java SDK.
+* You have Apache Maven {maven-version} installed.
+* You have access to an OpenShift cluster and the latest compatible version of the `oc` tool installed.
+* You are working in the correct OpenShift project namespace.
+//update this guide to provide link to topic xref:proc_verifying-the-openshift-project-namespace_quarkus-openshift[Switching to the required {RHOSSHORT} project].
+
+== Build and deploy a {project-name} application on OpenShift in a single step
+
+You can trigger a build and deployment with a single command.
+With this one command, you can build your application locally, trigger a container image build, then apply the generated OpenShift resources automatically.
+
+The generated resources use a Kubernetes `Deployment`, but still make use of OpenShift-specific resources, such as `Route`, `BuildConfig`, and so on.
+
+:build-additional-parameters: quarkus deploy openshift
+include::{includes}/devtools/build.adoc[]
+:!build-additional-parameters:
```

Comment:
Experimentally, if enclosing the admonitions doesn't fix the issue, I recommend troubleshooting by commenting this out and retrying the build.

Reply:
This wasn't the issue. Closing.